docs/4. Product Features/Data Analytics Page.md

```
@@ -0,0 +1,91 @@
+### Time Range 
+
+<img width="" height="385" alt="Screenshot 2026-07-08 at 4 25 34 PM" src="https://github.com/user-attachments/assets/2e5b4d13-0248-40a6-b738-dd9ae8bcd50e" />
+
+
+1. Sets the active date range, and pulls the data for the given time period.
+2. Range type can be used to set active date ranges using various options such as 
+- Current date, current week, current month, current quarter etc
+- Last date, last week, last month, last quarter etc
+- Advanced such as specifying “last week” or “last quarter” or “last 3 months”
+- And more custom options 
+3. In the example the date range is set as 2026-04-08 ≤ col < 2026-07-08 meaning the dashboard currently includes all activity from April 8 up to, but not including, July 8, 2026. 
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)